rules/PoiGeneric.js

```diff
@@ -164,8 +165,17 @@ function additiveScore(map, e1, e2) {
 
     var reason = result.reasons;
 
-    var searchRadius = Math.max(exports.getSearchRadius(e1), 
-        exports.getSearchRadius(e2));
+    var t1 = e1.getTags().toDict();
+    var t2 = e2.getTags().toDict();
+
+    var searchRadius = Math.max(exports.getSearchRadius(e1), exports.getSearchRadius(e2));
+
+    var stopPositionThreshold = 0.7;
+    if (getRelatedTagsByThreshold("public_transport=stop_position", e1.getTags().toDict(), stopPositionThreshold).length > 0 &&
+        getRelatedTagsByThreshold("public_transport=stop_position", e2.getTags().toDict(), stopPositionThreshold).length > 0)
+    {
+      searchRadius = Math.min(exports.getSearchRadius(e1), exports.getSearchRadius(e2));
```

Comment:
The two cases I'm dealing with are railway stations compared to bus stops and railway stations compared to railway tram stops.  

For railway stations compared to bus stations, the special rule for stop positions is not needed and just using the min of the two search radii fixes the problem (although worth noting that all of the brazil/latvia data has the default circular error=15m, which is the value that ends up being used as the search radii...but maybe that's expected).

For railway stations vs railway tram stops, just using the min of the search radii wasn't enough to prevent latvia review 1) in the redmine ticket from happening, and that's why the stop position threshold rule was added.  Although, I'll admit I'm little confused on why that stop position threshold rule is needed after I added rail tram stops to the railway schema and modeled them roughly after bus stops.  It seems like I shouldn't need that threshold rule to account for tram stops.